app/utils/parseCacheControl.ts

```diff
@@ -0,0 +1,90 @@
+import { parse as parseCacheControlHeader } from './cache-control'
+import { getTimeToLive } from './getCacheAnalysis'
```

Comment:
this directionality doesn't make sense. you've got A importing B importing A. let's just move `getTimeToLive` into its own file too. and make sure to pull its tests into its own test file.